public/templates/albums/show.html

```diff
@@ -33,7 +33,7 @@ <h1 bo-text="album.title"></h1>
 
     <div class="stretch-to-bottom details-columns">
         <div class="right">
-            <img class="cover" pfm-src-loader="album.covers.normal" pfm-src-size="normal" />
+            <img colorbox pfm-href-loader="album.covers.normal"  class="cover" pfm-src-loader="album.covers.normal" pfm-src-size="normal" />
```

Comment:
This generates invalid markup - only `a` tags should include `href` tags. The `img` element should be wrapped in an `a` tag that links to the cover art.

For that matter, this and the other links being generated for Colorbox go to the normal-size version of the image. They should link to the full-size original.